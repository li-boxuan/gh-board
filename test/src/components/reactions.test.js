import renderer from 'react-test-renderer';
import Reactions from '../../../src/components/reactions';

jest.mock('../../../src/github-client', () => ({
  getGraphQLClient: jest.fn(() => {}),
}));

it('does not display number zero', () => {
  const stat = {
    THUMBS_UP: 0,
    THUMBS_DOWN: 0,
    LAUGH: 0,
    HOORAY: 0,
    CONFUSED: 0,
    HEART: 0,
  };
  const rendered = renderer.create(
    <Reactions stat={stat} />
  );
  expect(rendered.toJSON()).toMatchSnapshot();
});
