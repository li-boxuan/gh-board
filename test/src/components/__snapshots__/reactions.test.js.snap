// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`does not display number zero 1`] = `
Array [
  <button
    className="reaction-btn reaction-btn-default"
    disabled={true}
    onClick={[Function]}
    type="button"
  >
    👍
     
  </button>,
  <button
    className="reaction-btn reaction-btn-default"
    disabled={true}
    onClick={[Function]}
    type="button"
  >
    👎
     
  </button>,
  <button
    className="reaction-btn reaction-btn-default"
    disabled={true}
    onClick={[Function]}
    type="button"
  >
    😄
     
  </button>,
  <button
    className="reaction-btn reaction-btn-default"
    disabled={true}
    onClick={[Function]}
    type="button"
  >
    🎉
     
  </button>,
  <button
    className="reaction-btn reaction-btn-default"
    disabled={true}
    onClick={[Function]}
    type="button"
  >
    😕
     
  </button>,
  <button
    className="reaction-btn reaction-btn-default"
    disabled={true}
    onClick={[Function]}
    type="button"
  >
    ❤️
     
  </button>,
]
`;
